flask_jsonlocale.py

```diff
@@ -6,26 +6,26 @@
 
 
 class Locales(object):
-    def __init__(self, app):
+    def __init__(self, app, app_dir):
         self.app = app
-        self.init_app(app)
+        self.init_app(app, app_dir)
         self._messages = {}
-    
-    def init_app(self, app):
-        app.config.setdefault('MESSAGES_DIR', 'messages')
+
+    def init_app(self, app, app_dir):
```

Comment:
this is a backwards-incompatible change, which I would like to avoid as long as possible. Could you try to find out how to get this information run-time?

Reply:
Do you mean that the init_app function header (line 14) should stay like the original, i.e. not adding more parameters to it?

Reply:
More, the __init__ method should not have any additional parameters. If we do that change, we'll need to update all the apps users - they wouldn't be just able to use what they do now, would have to manually pass the new parameter. 

It would be better to determine that live, so users don't need to pass that path.